### PR TITLE
Fix: Dark & Light mode

### PR DESCRIPTION
**Pull request by anmol-fzr**
Fix: homepage having only light header and rest page in dark mode, now all adapts to one theme either light or dark

#### Description

- Issue: #49
- Before this PR the studiocms homepage has mixed theme, Hero component has light mode and rest page has dark mode, with this PR whole page adapts to either light or dark mode

### CODE REVIEW

File: www/web/tailwind.config.mjs [hunk 1]
```diff
@@ -29,4 +29,5 @@ export default {
     },
   },
   plugins: [require('tailwind-scrollbar')],
+  darkMode: "class"
```
Comment (Adammatthiesen) [inline www/web/tailwind.config.mjs:32]:
```suggestion
  darkMode: "selector"
```
The selector strategy replaced the class strategy in Tailwind CSS v3.4.1.
src: https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually

File: www/web/src/components/FeatureCard.astro [hunk 2]
```diff
@@ -13,11 +13,11 @@ const { icon, description, feature } = Astro.props
 <div
   class='animate-border rounded-lg border border-transparent [background:linear-gradient(45deg,#172033,theme(colors.slate.800)_50%,#172033)_padding-box,conic-gradient(from_var(--border-angle),theme(colors.slate.600/.48)_80%,_theme(colors.indigo.500)_86%,_theme(colors.indigo.300)_90%,_theme(colors.indigo.500)_94%,_theme(colors.slate.600/.48))_border-box]'
 >
-  <div class='h-full place-self-center rounded-lg bg-[--purple-900] p-10'>
-    <div class='flex items-center justify-center gap-2 pb-6'>
+  <div class='h-full place-self-center rounded-lg bg-[--slate-100] dark:bg-[--purple-900] p-10'>
```
Comment (at-the-vr) [inline www/web/src/components/FeatureCard.astro:16]: --slate-100 is not a defined variable in `globals.css` it should be updated to `bg-slate-100`

File: www/web/src/layouts/BaseLayout.astro [hunk 3]
```diff
@@ -6,7 +6,7 @@ import Footer from '../components/Footer.astro'
 ---
 
 <html
-  class='scrollbar scrollbar-thumb-white scrollbar-track-[#1a172e] h-32 overflow-y-scroll'
+  class='scrollbar scrollbar-thumb-white scrollbar-track-[#1a172e] h-32 overflow-y-scroll dark'
```
Comment (at-the-vr) [inline www/web/src/layouts/BaseLayout.astro:9]: dark in itself is just a variant for tailwind, best to remove this as its not doing anything

File: www/web/src/styles/global.css [hunk 4]
```diff
@@ -7,11 +7,12 @@
     --purple-900: #05040b;
     --blob-dark: #9287ee;
     --blob-light: #201b4b;
+    --slate-100: #f1f5f9;
   }
 
   body {
-    @apply bg-[--purple-900] text-white dark:bg-[--purple-900] dark:text-slate-100;
-    font-family: 'zenkaku';
+    @apply bg-[--slate-100] text-[--purple-900] dark:bg-[--purple-900] dark:text-[--slate-100];
```
Comment (at-the-vr) [inline www/web/src/styles/global.css:14]: similar suggestion `bg-slate-100 text-[--purple-900] dark:bg-[--purple-900] dark:text-slate-100;`

**Review by at-the-vr — COMMENTED** [review]
Hey Anmol 👋 Great start with this PR 💯 I have left some comments for you to check whenever your free 😉. Definitely a brain teaser when considering a `Theme Selector` as well. Incase you are facing any difficulties, do share them here

**Pull request closed without merging**